seedcase_sprout/core/sprout_checks/data/get_pandera_checks.py

```diff
@@ -0,0 +1,152 @@
+import pandera.polars as pa
+from xmlschema.names import (
+    XSD_BASE64_BINARY,
+    XSD_DATE,
+    XSD_DATETIME,
+    XSD_DURATION,
+    XSD_GYEAR,
+    XSD_GYEAR_MONTH,
+    XSD_TIME,
+)
+
+from seedcase_sprout.core.properties import FieldProperties
+from seedcase_sprout.core.sprout_checks.data.data_type_checks import (
+    check_email,
+    check_geopoint,
+    check_json,
+    check_uuid,
+    check_xml_type,
+)
+
+# https://datapackage.org/standard/table-schema/#boolean
+BOOLEAN_VALUES = {"false", "False", "FALSE", "0", "true", "True", "TRUE", "1"}
+
+STRING_FORMAT_CHECKS = {
+    "email": pa.Check(
+        check_email,
+        element_wise=True,
+        error="Invalid email address.",
```

Comment:
```suggestion
        error="The value {value} given for the email field doesn't seem to be a correct email string.",
```

Is it possible to add the value so the message is more helpful?

Reply:
This will be used by Pandera to produce error messages like:
```log
E               pandera.errors.SchemaErrors: {
E                   "DATA": {
E                       "DATAFRAME_CHECK": [
E                           {
E                               "schema": null,
E                               "column": "my_string",
E                               "check": "Invalid email address.",
E                               "error": "Column 'my_string' failed validator number 0: <Check check_email: Invalid email address.> failure case examples: [{'my_string': 'jane_55@doe.c o.uk'}]"
E                           }
E                       ]
E                   }
```
So it already has this `failure case examples` field.
Should I still update the error messages to the longer ones?

Reply:
Ooo, gotcha! Ok, then forget about the `{value}` bit. But yes, we should try to make nicer error messages as much as is possible. I've updated my suggestions.

Reply:
Okay, thanks! I'll update them! 
But just for context, these messages will appear after all the data has been checked, so, for a large dataset, there will probably be several errors? As above, there will be one error message and then a list of failure cases. So I'm just wondering if saying "the given value" is not a bit confusing? But maybe it's obvious that it refers to each of the listed values :sweat_smile: .

But I think we shouldn't name the field as "the email field" because it's just the format, they can call it whatever they want (e.g. `contact_address`).

Reply:
How about something like: 
The given value doesn't seem to be formatted correctly as an email address.
?

Reply:
Yea, that wording works!

Reply:
The important thing here is that we try to make friendly and helpful error messages, not vague and overly concise ones. We want to prevent the proliferation of these types of error messages! :stuck_out_tongue: